### PR TITLE
{tools}[system/system] git-lfs v3.4.0

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ generoso

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ jsc-zen2

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/g/git-lfs/git-lfs-3.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+builddependencies = [('Go', '1.21.2')]
+
+files_to_copy = [(['bin/%(name)s'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/git-lfs'],
+    'dirs': [],
+}
+
+sanity_check_commands = [('git-lfs', '--version')]
```
Comment (boegel) [inline easybuild/easyconfigs/g/git-lfs/git-lfs-3.4.0.eb:26]:
```suggestion
sanity_check_commands = ["git-lfs --version"]

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3618.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/boegel/6bc8a8053339ad246174eab665a4a961 for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**